### DISCUSSION TITLE
Write raw cell data without any column caption on the first row

### DISCUSSION THREAD

**Discussion by eng-marani** [mini-software/MiniExcel#589] [Q&A] [answered]
I would like to simply write raw data (from an array string[][]) into an excel file.
All overload methods require a named column to be defined.
Do exist any way to write raw data starting from A1 column ? Any example available ?

**Comment by michelebastione** [comment, marked as answer]
Hello, sorry for the late reply. If you're still trying to solve the issue you can convert your data to a `DataTable` as a workaround:

```c#
string[][] data = [["a", "b", "c"], ["1", "2", "3"]];
var maxLen = data.Max(x => x.Length);
 
var dt = new DataTable();
 
for (int i = 0; i < maxLen; i++) 
    dt.Columns.Add(null, typeof(string));
 
foreach (var row in data) 
    dt.Rows.Add([..row]);
 
var conf = new OpenXmlConfiguration { AutoFilter = false, FreezeRowCount = 0 };
MiniExcel.SaveAs(yourPath, dt, printHeader: false, configuration: conf);
```
